docs/aspects.md

```diff
@@ -0,0 +1,32 @@
+Aspects & Task
+============================
+
+Together with a panel of colleagues with diverse expertese, we identified a number of aspects which are important when selecting LLMs for different use cases.
```

Comment:
Typo for Expertese

Reply:
Maybe also good to explain why we made the decision to score an aspect and use categories?